src/components/RegistrationForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react'
+import { Redirect, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { userActions } from '../actions/userActions'
+
+class RegisterForm extends Component {
+    state = {
+        user: {
+            firstName: "",
+            secondName: "",
+            email: "",
+            password: "",
+            confirmedPassword: "",
+            birthday: "",
+            isEmployee: false
+        },
+        submitted: false,
+        isSamePasswords: false,
+
+    };
+    handleInputChange = (event) => {
+        const target = event.target;
+        const value = target.type === "checkbox" ? (target.checked ? target.checked : false) : target.value;
+        const name = target.name;
+        const { user } = this.state;
+
+        this.setState({
+            user: {
+                ...user,
+                [name]: value
+            }
+        });
+    }
+    handlesubmit = (e) => {
+        e.preventDefault();
+
+
+        this.setState({ submitted: true });
+        this.setState({ submitted: true });
```

Comment:
Do you really need to have here two equal this.setState({}) calls?

Reply:
Done